### ISSUE TITLE
What is the maintenance status of this project?

### ISSUE THREAD

**Issue by williammartin** [99designs/keyring#138]
## Description

In [`gh`](https://github.com/cli/cli), we've had a long outstanding [PR](https://github.com/cli/cli/pull/7743) to use this module for secure credential management.

Looking at the repo commit history, community interactions (or lack thereof), and [dependent forks](https://github.com/99designs/go-keychain), it's not clear to me what the maintenance status of this project is.

I totally understand that ongoing maintenance of a project like this is burdensome, and might not be something 99designs or individuals want to continue with, but I do hope y'all (maybe @mtibben?) can offer some guidance on what potential consumers of this project might be able to expect in the short term or long term?

Cheers!

**Comment by ghthor** [comment]
I'm really looking forward to the gh-cli storing my non-expiring GH token inside a password store on linux. I hope we hear back soon on this question.

**Comment by williammartin** [comment]
@ghthor sorry if you already responded but I couldn't spot your name in any of the existing threads. What about the current approach doesn't work for you on Linux? The primary intention of moving over to this project was to avoid providing excess privileges to the `security` binary on MacOS.

**Comment by ghthor** [comment]
@williammartin On linux the GH Token secret is stored in plaintext on disk

**Comment by williammartin** [comment]
`gh` tries to store the token in your keyring using https://github.com/zalando/go-keyring/blob/b38ab4b7224a1f2d243fc35cd3a646ad1cdcdf40/keyring_unix.go#L17 on linux. There's no blocker, requiring us to use this library. Feel free to file an issue on `cli/cli` explaining your setup and hopefully we can help you out.

**Comment by mbevc1** [comment]
Maintained fork: https://github.com/ByteNess/keyring/

**Comment by lox** [comment]
As the original author of this, I've also got a fork going at https://github.com/lox/keyring. Trying to encourage the 99designs folks to do something about this.

**Comment by mbevc1** [comment]
Cool, we got quite a few new backends merged to ByteNess/keyring . Feel free to check it out

**Comment by lox** [comment]
I've reached out to the 99designs folks to see if they can either transfer or archive the repo. /cc @lwc

**Comment by mbevc1** [comment]
Yeah, it'd be useful to at least call out active forks. I managed to get this to aws-vault repo
